.github/workflows/code-checks.yml

```diff
@@ -13,6 +13,7 @@ concurrency:
 jobs:
   mypy:
     runs-on: ubuntu-20.04
+    # TODO: package parametrization
```

Comment:
```suggestion
```
Why this shall be parametrized?

Reply:
I can imagine that there are mypy issues after rewriting the imports